package.json

```diff
@@ -19,26 +26,153 @@
     "url": "https://github.com/auth0/auth0-logs-to-logentries/issues"
   },
   "homepage": "https://github.com/auth0/auth0-logs-to-logentries#readme",
+  "auth0-extension": {
+    "externals": [
+      "async@2.1.2",
+      "auth0-extension-express-tools@1.1.5",
```

Comment:
I think this should be bumped to `@1.1.6`

Reply:
`auth0-extension-express-tools@1.1.6` now included into extension